server/remote/msgraph/call.go

```diff
@@ -34,6 +35,7 @@ func (c *client) Call(method, path string, in, out interface{}) (responseData []
 		if err != nil {
 			return nil, err
 		}
+		inBody = buf
```

Comment:
What was this `inBody` variable used for before if it wasn't getting assigned?

Reply:
@mickmister, it wasn't being used.  The `Call()` method was written but not yet thoroughly tested and the use cases didn't actually leverage this code.  An example would be a call to this function that didn't require a JSON payload.  

I ran into an issue when using is this function for creating a calendar event which is a POST method. Other calls to the function, thus far, are GET methods and didn't require a JSON Payload.

Reply:
Was my bad - never tested this path, it's all POC. Jason discovered it trying to use .Call